osquery/database/database.cpp

```diff
@@ -76,7 +77,8 @@ Status serializeRowJSON(const Row& r, std::string& json) {
   try {
     pt::write_json(output, tree, false);
   } catch (const pt::json_parser::json_parser_error& e) {
-    // The content could not be represented as JSON.
+    LOG(WARNING) << "Error serializing row as JSON: Database (" << e.what()
```

Comment:
I don't think you need the "Database" for these, each LOG will be decorated with the file name and line number.